packages/schema/parse/src/extract/imported-module-types.ts

```diff
@@ -73,7 +77,7 @@ const visitorEnter = (
       map: def
         ? ({ ...def, name: node.name.value } as MapDefinition)
         : undefined,
-      required: def && def.required ? true : false,
+      required: def && def.required ? true : undefined,
```

Comment:
It seems kinda weird to me that `required` is being `true` or `undefined` - IMO we should have it true or false as before; no strong opinion on this just sharing my thoughts

Reply:
It's much better to keep it undefined then the false because it takes a lot of space to store `required: false`